force-app/main/default/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -2651,11 +2651,12 @@ export default class GeFormRenderer extends LightningElement{
             ? this.CUSTOM_LABELS.geTextUpdating
             : this.CUSTOM_LABELS.geTextSaving;
         delete dataImportFromFormState[apiNameFor(PAYMENT_AUTHORIZE_TOKEN)];
+        const dataImportfieldsInfo = this.dataImportObjectInfo?.data?.fields || {};
         Object.keys(dataImportFromFormState).forEach((field)=>{
-            if(dataImportFromFormState[field] === undefined) {
```

Comment:
@salesforce-suyash-more Don't we need to make sure Boolean fields are True or False? So if they are undefined or null, they should be set to False? Maybe that happens elsewhere.

Reply:
That's correct, I beleive it's better to set it to true/false instead of null since there is no concept empty on checkbox